webpack/resolve-chunk-plugin/src/ResolveChunkPlugin.ts

```diff
@@ -0,0 +1,70 @@
+/// <reference path="./webpackTypes.d.ts" />
+
+import * as Webpack from 'webpack';
+
+// tslint:disable-next-line:variable-name
+const ConstDependency: IConstDependency = require('webpack/lib/dependencies/ConstDependency');
+
+export class ResolveChunkPlugin implements Webpack.Plugin {
+  public apply(compiler: Webpack.Compiler): void {
+    compiler.plugin('compilation', (compilation, data) => {
+      const chunkIdMap: Map<string, ((id: number | undefined) => void)[]> =
```

Comment:
>((id: number | undefined) => void [](start = 36, length = 33)

this could be upgraded to a named type

Reply:
I don't think that'd provide much value.

---
In reply to: [139566050](https://github.com/Microsoft/web-build-tools/pull/375#discussion_r139566050) [](ancestors = 139566050)